src/main/kotlin/day05/Collection.kt

```
@@ -0,0 +1,40 @@
+}
+
+class Man(
+    var name: String,
+    val hp: Int
+) {
+
+}
+
+class Collection {
+
+}
```

detekt (1.23.7)

[warning] The class or object Collection is empty.

(detekt.empty-blocks.EmptyClassBlock)